en/config/safety.md

```diff
@@ -0,0 +1,225 @@
+
+![Safety - Battery (QGC)](../../images/qgc/setup/safety_battery.png)
+
+The most common configuration is to set the values and action as above (with `Warn > Failsafe > Emergency`).
+With this configuration the failsafe will trigger warning, then return, and finally landing if capacity drops below the respective levels.
+ 
+It is also possible to set the *Failsafe Action* to warn, return, or land. 
+In this case the vehicle will perform the specified action when the *Warn* failsafe level is reached.
+
+The settings and underlying parameters are shown below.
+  
+Setting | Parameter | Description
+--- | --- | ---
+Failsafe Action | [COM_LOW_BAT_ACT](../advanced_config/parameter_reference.md#COM_LOW_BAT_ACT) | Warn, Return, or Land based when capacity drops below Battery Warn Level, OR Warn, then return, then land based on each of the level settings below. 
+Battery Warn Level | [BAT_LOW_THR](../advanced_config/parameter_reference.md#BAT_LOW_THR) | Percentage capacity for warnings (or other actions).
+Battery Failsafe Level | [BAT_CRIT_THR](https://docs.px4.io/en/advanced_config/parameter_reference.html#BAT_CRIT_THR) | Percentage capacity for Return action.
+Battery Emergency Level | [BAT_EMERGEN_THR](../advanced_config/parameter_reference.md#BAT_EMERGEN_THR) | Percentage capacity for triggering Land (immediately) action.
+
+
+### RC Loss Failsafe
+
+The RC Loss failsafe is triggered if the RC transmitter link is lost.
+
+![Safety - RC Loss (QGC)](../../images/qgc/setup/safety_rc_loss.png)
+
+The settings and underlying parameters are shown below.
+
+Setting | Parameter | Description
+--- | --- | ---
+RC Loss Timeout | [COM_RC_LOSS_T](../advanced_config/parameter_reference.md#COM_RC_LOSS_T) | Amount of time after losing the RC connection before the failsafe will trigger.
+Failsafe Action | [NAV_RCL_ACT](../advanced_config/parameter_reference.md#NAV_RCL_ACT) | Disabled, Loiter, Return, Land, RC Auto Recovery, Terminate, Lockdown.
+RC Loss Loiter Time | [NAV_RCL_LT](../advanced_config/parameter_reference.md#NAV_RCL_LT) | If the *Failsafe Action* (`NAV_RCL_ACT`) is set to *CASA Outback Challenge rules* this sets the loiter time after RC loss.
+
+<!-- note, does not seem any way to set NAV_RCL_LT in qgc -->
+
+
+### Data Link Loss Failsafe
+
+The Data Link loss failsafe is triggered if a telemetry link is lost.
+
+![Safety - Data Link Loss (QGC)](../../images/qgc/setup/safety_data_link_loss.png)
+
+The settings and underlying parameters are shown below.
+
+Setting | Parameter | Description
+--- | --- | ---
+Data Link Loss Timeout | [COM_DL_LOSS_T](../advanced_config/parameter_reference.md#COM_DL_LOSS_T) | Amount of time after losing the data connection before the failsafe will trigger.
+Failsafe Action | [NAV_DLL_ACT](../advanced_config/parameter_reference.md#NAV_DLL_ACT) | Disabled, Loiter, Return, Land, RC Auto Recovery, Terminate, Lockdown.
+
+
```

Comment:
There are a bunch of other parameters that appear to be related here: https://docs.px4.io/en/advanced_config/parameter_reference.html#data-link-loss

When do these come into play? - what vehicles?

Reply:
These are used in combination with 'Data Link Auto Recovery (CASA Outback Challenge rules)'